### PR TITLE
WRR-387: `Panels`: Updated documentation for arranger

### PR DESCRIPTION
**Pull request by vJIYEv**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Default value of `arranger` prop in `Sandstone/Panels` does not match the documentation.
Documentation says the default is `ui/ViewManager.SlideLeftArranger`, but actual value is `BasicArranger` from `internals/Panles/Arrangers.js`.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Default value was changed in https://github.com/enactjs/sandstone/pull/315 from `SlideLeftArranger` to `BasicArranger` and `arrange` in `BasicArranger` was changed to `deferArrange` in https://github.com/enactjs/sandstone/pull/577
But `BasicArranger` is for internal use.
So instead of updating documentation of `arranger` in `Sandstone/Panels`, I updated documentation of `BasicArranger` to explain what it is.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRR-387

### Comments
Enact-DCO-1.0-Signed-off-by: Jiye Kim (jiye.kim@lge.com)

### CODE REVIEW

File: internal/Panels/Arrangers.js [hunk 1]
```diff
@@ -129,6 +129,9 @@ const deferArrange = (config, keyframes, options) => {
 /**
  * Arranger that slides panels in from the right and out to the left.
  *
+ * This arranger is visually same as {@link ui/ViewManager.SlideLeftArranger} when transition but uses transform percentages instead of pixel values.
```
Comment (MikyungKim) [inline internal/Panels/Arrangers.js:132]: It looks like `ui/ViewManager.SlideLeftArranger` already use percentages. So I think this comment should be revised.
If the visual is the same, what makes difference between `SlideLeftArranger` and `BasicArranger`?
I see BasicArranger uses `deferArrange` and `offset`. Please find more details about this.

Comment (vJIYEv) [reply internal/Panels/Arrangers.js]: 1. `offset` values here don't affect animation (https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API/Keyframe_Formats)
2. `deferArrange` was defined in https://github.com/enactjs/sandstone/pull/577 and it prevents skipping transition animations under system load. I updated the comments.

File: internal/Panels/Arrangers.js [hunk 2]
```diff
@@ -129,6 +129,9 @@ const deferArrange = (config, keyframes, options) => {
 /**
  * Arranger that slides panels in from the right and out to the left.
  *
+ * This arranger is visually same as {@link ui/ViewManager.SlideLeftArranger} when transition but prevents skipping transition animations under system load.
+ * Because BasicArranger is an improved version of {@link ui/ViewManager.SlideLeftArranger}, {@link ui/ViewManager.SlideLeftArranger} can be written in documentation instead of BasicArranger in unavoidable case.
```
[the remark this answers is not in the document]
Comment (vJIYEv) [reply internal/Panels/Arrangers.js]: ok

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**